# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/cache/DBSchema.h
# Commit: 0287f5604a69
# Full Hash: 0287f5604a69c65705743257a5bbe24a5bdb9d9f
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2020-12-14 23:03:37
# Regressor Bug: 1678030
# File Overlap Count: 1
# Description:
#   Bug 1678030 - Move CollectWhileHasResult to QuotaCommon and use in other quota clients. r=dom-workers-and-storage-reviewers,ttung
#   
#   This patch also:
#   
#   - introduces CollectElementsWhileHasResult(Typed) to collects elements in and
# ==============================================================================

diff -r 6dd8d66f8116 -r 0287f5604a69 dom/cache/DBSchema.h
--- a/dom/cache/DBSchema.h	Thu Dec 10 11:45:42 2020 +0000
+++ b/dom/cache/DBSchema.h	Mon Dec 14 10:42:36 2020 +0000
@@ -39,7 +39,7 @@
 Result<DeletionInfo, nsresult> DeleteCacheId(mozIStorageConnection& aConn,
                                              CacheId aCacheId);
 
-Result<nsTArray<CacheId>, nsresult> FindOrphanedCacheIds(
+Result<AutoTArray<CacheId, 8>, nsresult> FindOrphanedCacheIds(
     mozIStorageConnection& aConn);
 
 Result<int64_t, nsresult> FindOverallPaddingSize(mozIStorageConnection& aConn);